standards/request-response.md

```diff
@@ -527,6 +527,47 @@ Sps-Execution-Context:                  // values must be at least a character l
 Sps-Execution-Context: 1                // valid, but SHOULD be human-readable.
 ```
 
+#### Sps-User-Agent
+**Type**: Request
+
+**Support**: OPTIONAL
+
+**Description**: 
+`Sps-User-Agent` is a direct augmentation of `User-Agent` and is intended to be paired with it, following
+the format of the SPS [Authentication and Authorization Guardrail](https://atlassian.spscommerce.com/wiki/display/Guardrails/[Authentication+and+Authorization).
+Typically, an API should require the `User-Agent` header, and that it **SHOULD** follow the format 
+`serviceName[/serviceVersion] [sdk[/sdkVersion ](serviceTechRegistryId)` to include the valid tech-registry
+serviceName and serviceId of the client.
+However, is common for some clients, such as web-browsers, to lock a client out of setting the `User-Agent`.
+Thus, `User-Agent` cannot reliably encode actually identifying information for general applications.
+Furthermore, auth tokens also do not include this information, and it is also common to forward a user_token, where
+what an API would rather know is not what "user" but what "client" is making the request.
```

Comment:
Could be a bit wordy, but this is my attempt to explain the long causality that causes one to want this header in the critical API space. 

parcel-service and NCS already support a custom header like this, which is `X-User-Agent-Override`, and we find it very valuable for understanding UI behavior.

One thing this doesn't quite call out is that I think any service that implements this header then should require the correct format for `User-Agent` as well.  It's more like a cascade.
1. Check `Sps-User-Agent` first, set as Agent value.
2. If `Sps-User-Agent` not present, set `User-Agent` as Agent value.
3. If Agent value is invalid format, reject with a `400`

Just some higher level thought about this, but there's also opportunities here possibly for tooling like atlas service mesh to populate this for some requests to give apps a better automatic user-agent.